NMakefile

```diff
@@ -14,18 +14,21 @@ CARGO=cargo
 !ENDIF
 
 # Define build target and flags
+# --target=x86_64-win7-windows-msvc
 !IFDEF DEBUG
 TARGET=target\debug\pep_engine_sequoia_backend.dll
+TARGET_DIR=target\debug
 CARGOFLAGS=
 !ELSE
 TARGET=target\release\pep_engine_sequoia_backend.dll
+TARGET_DIR=target\release
 CARGOFLAGS=--release
 !ENDIF
 
 build:
-	$(CARGO) build --features crypto-botan --no-default-features $(CARGOFLAGS)
```

Comment:
Is `--features crypto-botan` not important for the build to ensure we use Botan as the crypto backend? In this PR, it vanishes.